### DISCUSSION TITLE
SkyOS operating system choice

### DISCUSSION THREAD

**Discussion by pineapplee0** [skylerspaeth/SkyOS#6] [Q&A] [answered]
What made you switch over to using Fedora when the original SkyOS was in Debian?

**Comment by skylerspaeth** [comment, marked as answer]
Hi pineapple, thank you for your interest in SkyOS!

I decided to switch to Fedora due to it having more up-to-date packages compared to Debian. Debian is a wonderful and stable OS, but not having compatible NVIDIA drivers and NVENC-enabled OBS packages readily available on the built-in package manager caused more trouble than it was worth. Fedora has been great thus far and I hope you find the same once SkyOS has been released!
